.travis.yml

```
@@ -139,6 +139,24 @@ matrix:
       - tools/setup_android.sh
      script:
       - tools/build_android.sh --release --verbose
+   - name: "Cross build x86 from x86_64"
+     language: rust
+     rust: stable
+     env:
+       TARGET=i686-unknown-linux-gnu
+     before_install:
+      # Install and use the current stable release of Go
+      - gimme --list
+      - eval "$(gimme stable)"
+      - gimme --list
+      - rustup target add $TARGET || true
```

Why is `|| true` needed?

Also might want to add a comment here, like:
```suggestion
      # Install x86 Rust target for cross-building.
      - rustup target add $TARGET || true
```